core-java-modules/core-java-string-algorithms-3/src/test/java/com/baeldung/repeatedcharstring/RepeatedCharacterStringUnitTest.java

```diff
@@ -0,0 +1,95 @@
+package com.baeldung.repeatedcharstring;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+
+import static ch.qos.logback.core.CoreConstants.EMPTY_STRING;
+import static java.util.stream.Stream.generate;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
There's no need for this lifecycle. Let's remove it.

Reply:
Removed